File: challenges/2-tabuada/ruby/yosh1tsune/tabuada.rb
puts "\n================ Desafio Tabuada ================"

begin
  puts "\nInsira até qual número você deseja ver a tabuada: "
  quantidade = gets.chomp.to_i
end while quantidade <= 0 
tabuada = 1

while tabuada <= quantidade
  multiplicador = 1
  puts "\n===== Tabuada do #{tabuada} =====\n\n"
  while multiplicador <= 10
    puts "  #{tabuada}   *   #{multiplicador}   =   #{tabuada * multiplicador}"
    multiplicador = multiplicador + 1
  end
  puts "\n========================\n\n"
  tabuada = tabuada + 1
end

File: challenges/2-tabuada/c/levxyca/tabuada.c
#include <stdio.h>

int main() {
    int numero;

    printf("\n::::::::::::: TABUADA:::::::::::::\n");

    printf("Você deseja ver a tabuada até qual número?\n");
    scanf("%d", &numero);

    for(int i = 1; i <= numero; i++) {
        printf("\n~===========================~\n");
        printf("A tabuada do número %d é:\n", i);
        for(int j = 1; j <= 10; j++) {
            printf("%d * %d = %d\n", i, j, i*j);
        }
        ("\n~===========================~\n");
    }
    return 0;
}